cc.config.json

```diff
@@ -47,9 +36,6 @@
         "2d": {
             "modules": ["2d", "sorting"],
             "dependentAssets": [
-                "60f7195c-ec2a-45eb-ba94-8955f60e81d0",
-                "1c02ae6f-4492-4915-b8f8-7492a3b1e4cd",
-                "810e96e4-e456-4468-9b59-f4e8f39732c0",
```

Comment:
"60f7195c-ec2a-45eb-ba94-8955f60e81d0" is "builtin-sprite.effect", unused
"1c02ae6f-4492-4915-b8f8-7492a3b1e4cd" is "builtin-graphics.effect", unused
"810e96e4-e456-4468-9b59-f4e8f39732c0" is "builtin-clear-stencil.effect", unused